internal/engine/rtengine/engine.go

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package engine
+// Package rtengine contains the rule type engine
+package rtengine
```

Comment:
**suggestion (non-blocking):** I would name it simply `rules`, this package is under `internal/engine` anyway.

Reply:
@blkt Attempting to name this package was a pain. The problem is that `rules`, `rule`, `ruletype` and `ruletypes` are common variable names in the code which import this package. `ruleengine` was considered, but that is just annoying to type :)